codegens/r-httr/lib/rHttr.js

```diff
@@ -0,0 +1,300 @@
+const getOptions = require('./options').getOptions,
+  sanitizeString = require('./util/sanitize').sanitizeString,
+  sanitizeOptions = require('./util/sanitize').sanitizeOptions,
+  parseBody = require('./util/parseBody').parseBody;
+
+/**
+  * Returns the snippet header
+  *
+  * @module convert
+  *
+  * @returns {string} the snippet headers (uses)
+  */
+function getSnippetHeader () {
+  return 'library(httr)\n';
+}
+
+/**
+  * Returns the snippet footer
+  *
+  * @module convert
+  * @returns {string} the snippet headers (uses)
+  */
+function getSnippetFooter () {
+  return 'cat(content(res, \'text\'))';
+}
+
+/**
+ * Gets the defined indentation from options
+ *
+ * @param  {object} options - process options
+ * @returns {String} - indentation characters
+ */
+function getIndentation (options) {
+  if (options && options.indentType && options.indentCount) {
+    let charIndentation = options.indentType === 'Tab' ? '\t' : ' ';
+    return charIndentation.repeat(options.indentCount);
+  }
+  return '  ';
+}
+
+/**
+ * Used to get the headers and put them in the desired form of the language
+ *
+ * @param  {Object} request - postman SDK-request object
+ * @returns {String} - request headers in the desired format
+ */
+function getRequestHeaders (request) {
+  return request.headers.members;
+}
+
+/**
+ * Returns the request's url in string format
+ *
+ * @param  {Object} request - postman SDK-request object
+ * @returns {String} - request url in string representation
+ */
+function getRequestURL (request) {
+  return request.url.toString();
+}
+
+/**
+  * Validates if the input is a function
+  *
+  * @module convert
+  *
+  * @param  {*} validateFunction - postman SDK-request object
+  * @returns {boolean} true if is a function otherwise false
+  */
+function validateIsFunction (validateFunction) {
+  return typeof validateFunction === 'function';
+}
+
+/**
+ * Returns the request's url in string format
+ *
+ * @param  {Object} request - postman SDK-request object
+ * @returns {String} - request url in string representation
+ */
+function getRequestMethod (request) {
+  return request.method;
+}
+
+/**
+ * Transforms an array of headers into the desired form of the language
+ *
+ * @param  {Array} mapToSnippetArray - array of key values
+ * @param  {String} indentation - used for indenting snippet's structure
+ * @param  {boolean} sanitize - whether to sanitize the key and values
+ * @returns {String} - array in the form of [ key => value ]
+ */
+function getSnippetArray (mapToSnippetArray, indentation, sanitize) {
+  // mapToSnippetArray = groupHeadersSameKey(mapToSnippetArray);
```

Comment:
You may forget to add the group headers or this commented line is no longer needed